message DseBlackListOpt
{
	optional int32	res = 1;
	optional int32 type = 2;//1加入 2移除
	optional string uid	= 3;
}